jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -466,6 +468,12 @@ public Object configure() throws Exception
                     throw new IllegalStateException(String.format("No matching constructor %s in %s", oClass, _configuration));
                 }
             }
+            else
+            {
+                // The Object already existed, if it has <Arg> nodes, warn about them not being used.
+                XmlConfiguration.getNodes(_root, "Arg")
+                    .forEach((node) -> LOG.warn("Ignored arg {} in {}", node, this._configuration._location));
```

Comment:
I think we should somehow increment the index variable so that the Arg is ignored